spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -23,15 +23,15 @@
       attachment2: upload_2,
      'attachment-info' => <<-eojson
         {
-          'attachment2': {
-            'filename': 'photo2.jpg',
-            'name': 'photo2.jpg',
-            'type': 'image/jpeg'
+          "attachment2": {
+            "filename": "photo2.jpg",
+            "name": "photo2.jpg",
+            "type": "image/jpeg"
           },
-          'attachment1': {
-            'filename': 'photo1.jpg',
-            'name': 'photo1.jpg',
-            'type': 'image/jpeg'
+          "attachment1": {
+            "filename": "photo1.jpg",
+            "name": "photo1.jpg",
```

Comment:
It might make sense to revert these all back to single quote. It appears there is a lot in this file that is still single quotes. If there is a plan to update this over time, then how does just updating your code for now to double quotes and reducing the noise?